connector_search_engine/models/se_binding.py

```diff
@@ -197,21 +196,20 @@ def _recompute_json(self, force_export=False):
                 error = self._validate_record(work, index_record)
                 if error:
                     validation_errors.setdefault(error, []).append(binding.id)
-                    to_be_checked.append(binding.id)
                     # skip record
-                    continue
                 if binding.data != index_record or force_export:
                     vals = {"data": index_record}
                     if binding.sync_state != "to_update":
                         vals["sync_state"] = "to_update"
                         vals["date_modified"] = fields.Datetime.now()
+                    if error:
+                        vals["sync_state"] = "to_be_checked"
+                        vals["__error__"] = error
```

Comment:
Of course it would be better to have a specific field. However:
* we could have tons of records on different models
* in a pre-migrate step you don't have at hand all the models that could inherit from the mixin as they are not loaded yet
* we cannot safely add the field via sql to all the tables that would need it

Or, at least, I don't have a smart way to do it... 
This is better than nothing for now :)